node/node.go

```diff
@@ -430,6 +430,14 @@ func (n *Node) startRPC() error {
 	}
 
 	initAuth := func(port int, secret []byte) error {
+		authModules := DefaultAuthModules
+		for _, module := range n.config.HTTPModules {
+			if module == "miner" {
+				authModules = append(authModules, "miner")
+				break
+			}
+		}
```

Comment:
`if slices.Contains(...) { ... }` might be more readable here

Reply:
Fixed! I have not done Go in a while.